### DISCUSSION TITLE
Including other resources in a native executable

### DISCUSSION THREAD

**Discussion by RF-Flores** [quarkusio/quarkus#45251] [Q&A]
I attempting to add a file and access it (unsuccessfully) in my native executable.

I am following [this](https://quarkus.io/guides/writing-native-applications-tips#including-resources) official guide. The file i wish to include is located in src/main/resources/data directory.

I have added the following to the application.properties :

` quarkus.native.resources.includes=data/**` (it is not being overwritten by a qa profile property)

However, when the application is running it is unable to find the file with the following class loader paths:

```
this.getClass().getClassLoader().getResource("file.json");
this.getClass().getClassLoader().getResource("/data/file.json");
this.getClass().getClassLoader().getResource("src/main/resources/data/file.json");
```

I am building my native executable and pushing it to a registry directly with the following:

`./mvnw package -Dquarkus.profile=qa -Dnative -Dquarkus.native.container-build=true`

All of these are returning a null URL. Am i missing something or some detail?

[a comment by an automated account is not shown]

**Comment by zakkak** [comment]
@RF-Flores have you tried

```
this.getClass().getClassLoader().getResource("data/file.json");
```
?
